packages/insomnia-config/package.json

```
@@ -17,15 +17,14 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "bootstrap": "npm run generate",
     "lint": "eslint . --ext .js,.ts,.tsx --cache",
     "lint:fix": "npm run lint -- --fix",
     "test": "jest",
     "pregenerate": "npm run build",
     "generate": "ts-node ./src/generated/generate.ts",
     "clean": "tsc --build tsconfig.build.json --clean",
     "prebuild": "npm run clean",
-    "build": "tsc --build tsconfig.build.json"
+    "build": "tsc --build tsconfig.build.json && ts-node ./src/generated/generate.ts"
```

```suggestion
    "build": "tsc --build tsconfig.build.json && npm run generate"
```

let's reduce the margin for error here, at least, then.

---

that would create a loop since generate will pregenerate the build step again.

---

ok, so then we remove the "pregenerate"?